### PR TITLE
feat(workspace): desktop resource support eip_id field

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:
The workspace support the internet access function.
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
 make testacc TEST=./huaweicloud/services/acceptance/workspace TESTARGS='-run TestAccDesktop_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/workspace -v -run TestAccDesktop_basic -timeout 360m -parallel 4
=== RUN   TestAccDesktop_basic
=== PAUSE TestAccDesktop_basic
=== CONT  TestAccDesktop_basic
--- PASS: TestAccDesktop_basic (1245.40s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/workspace 1245.440s
```

### CODE REVIEW

File: huaweicloud/services/workspace/resource_huaweicloud_workspace_desktop.go [hunk 1]
```diff
@@ -178,6 +178,11 @@ func ResourceDesktop() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"eip_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```
Comment (Lance52259) [inline huaweicloud/services/workspace/resource_huaweicloud_workspace_desktop.go]: No `computed` behavior

File: docs/resources/workspace_desktop.md [hunk 2]
```diff
@@ -161,6 +166,8 @@ In addition to all arguments above, the following attributes are exported:
 * `data_volume` - The configuration of data volumes.
   The [object](#desktop_volume_attr) structure is documented below.
 
+* `eip_id` - This EIP ID is bound from the desktop.
```
Comment (Lance52259) [inline docs/resources/workspace_desktop.md]: `eip` just only argument, not attribute.

**Comment by wuzhuanhong** [comment]
make testacc TEST=./huaweicloud/services/acceptance/workspace TESTARGS='-run TestAccDesktop_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/workspace -v -run TestAccDesktop_basic -timeout 360m -parallel 4
=== RUN   TestAccDesktop_basic
=== PAUSE TestAccDesktop_basic
=== CONT  TestAccDesktop_basic
--- PASS: TestAccDesktop_basic (1268.72s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/workspace 1268.784s

**Pull request merged**